source/features/github-bugs.css

```diff
@@ -10,16 +10,19 @@
 	margin: -5px 0;
 }
 
-/* Reposition committer's avatar on PR commits #3444 */
-.pull-request-tab-content .commit .AvatarStack {
-	display: inline-block;
-	margin-bottom: -4px;
-	margin-right: -4px;
-}
-
 /* Add missing border radius to collapsed PR files #3444 */
 .js-blob-wrapper,
 .file:not(.open) .file-header.file-header--expandable {
 	border-bottom-left-radius: 5px;
 	border-bottom-right-radius: 5px;
 }
+
+/* Align author name on PR sticky header */
```

Comment:
This issue has been confirmed on the latest Chrome 86 (table build)

Reply:
Give it time. I don’t want more CSS code to fix something they probably will fix soon

Reply:
This issue has been there for a while, so does the misaligned dropdown (though the former one is only noticeable in newer versions of Chrome).
But I agree these fixes should not be hurried. Do we have some staging area for this kind of improvements (like an issue)?

Reply:
1. It's just on your computer

<img width="583" alt="" src="https://user-images.githubusercontent.com/1402241/96289760-33558900-0fab-11eb-9c71-252dd490451a.png">

2. This CSS affects everyone, not just Chrome users, so if this fixes it for you, it breaks everyone else’s

3. We don't even show the username anymore thanks to `clean-conversation-headers`, so this "fix" shouldn't be part of RG anyway in the first place.

Reply:
Is it possible that you have corrupted font files?

Reply:
> 1. It's just on your computer
> 

I just asked a fellow and he can also reproduce it. We are both on Windows, is that the problem?

I will test it on Linux later

Reply:
On Chrome 86 on Windows as well and it looks fine.

Reply:
Are you and your fellow on zh-CN locales? I don’t have experience with that, but I would guess that it might affect alignment.

Reply:
> Are you and your fellow on zh-CN locales?

You are right! The issue disappeared right after I changed my locale.

Then I have no problem with removing the fix. :+1:  I will keep it in my custom style.